gen/generator.toml

```diff
@@ -132,6 +132,9 @@ opaque_as_mutable_struct = true
 # if true, use Julia 1.5's new `@ccall` macro 
 use_ccall_macro = true
 
+# if true, variadic functions are wrapped with `@ccall` macro. Otherwise variadic functions are ignored.
+wrap_variadic_function = true
```

Comment:
Let's turn this feature off by default for now. We can turn it on again in the final v0.14 release.